packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -96,6 +101,21 @@ export interface OdspErrorResponse {
     };
 }
 
+/** Redirect error from ODSP  */
```

Comment:
```suggestion
/** Error encapsulating the error response from ODSP containing the redirect location when a resource has moved  */
```
Or something more explicit like this.